extensions/machine-learning-services/package.nls.json

```diff
@@ -7,6 +7,7 @@
 	"title.endpoints": "Endpoints",
 	"mls.command.managePackages": "Manage Packages in SQL Server",
 	"mls.command.manageLanguages": "Manage External Languages",
+	"mls.command.predictModel": "Make prediction",
```

Comment:
shouldn't we have this in localize() format

Reply:
this is the localization file for commands

---
In reply to: [389164680](https://github.com/microsoft/azuredatastudio/pull/9450#discussion_r389164680) [](ancestors = 389164680)

Reply:
ah my bad